pulp_ansible/app/models.py

```diff
@@ -345,7 +361,7 @@ class Meta:
         unique_together = ("namespace", "name")
 
 
-class AnsibleRepository(Repository):
+class AnsibleRepository(Repository, AutoAddObjPermsMixin):
```

Comment:
We also need to add a flag for private repositories. I know we have a separate issue for that, so if you want to tackle it in a different PR, that's totally fine